### DISCUSSION TITLE
ARM64 error now [Nov 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4420] [Discord Threads]
> 🙋‍♂️ **go4cas**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Alsu**  
#### Hello, is there solution to this? I am running into the same issue  
#  
> 👨‍💻 **pranavxc**  
#### <@955863278133870672>: can you share more details?  
#  
> 🙋‍♂️ **Alsu**  
#### I get exact same error when I am trying to build and run docker image using this script:  ```#!/usr/bin/env bash   
#### set -euxo pipefail   
#### # Build NocoDB image   
#### cd packages/nocodb-sdk   
#### npm install && npm run build   
#### cd ../..   
#### cd packages/nc-gui   
#### npm install   
#### targetEnv=PROD npm run build:copy   
#### cd ../..   
#### cd packages/nocodb   
#### cp -rf ../nocodb-sdk .   
#### cp -rf ../nc-lib-gui .   
#### npm install   
#### npm install faker@^4.1.0   
#### #NODE_OPTIONS=--openssl-legacy-provider    
#### npm run build   
#### #NODE_OPTIONS=--openssl-legacy-provider    
#### npm run docker:build   
#### docker build . -t nocodb ```  
#  
> 🙋‍♂️ **Alsu**  
#### my docker image list looks like this: ```REPOSITORY   TAG          IMAGE ID       CREATED        SIZE   
#### nocodb       latest       d756e017776c   19 hours ago   118MB   
#### <none>       <none>       330231037c24   19 hours ago   1.22GB   
#### <none>       <none>       7e17f93024b2   19 hours ago   1.59GB   
#### node         16           147af8719073   2 weeks ago    911MB   
####    
#### ```  
#  
> 🙋‍♂️ **Alsu**  
#### never mind, i was running docker from wrong folder.  
#  
> 👨‍💻 **pranavxc**  
#### <@955863278133870672> : not an issue. So finally it's working without any issue, right ?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/851824635721154590)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
